ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/V2.hs

```diff
@@ -180,8 +180,8 @@ mkInternals ::
 mkInternals bss h = TestInternals {
```

Comment:
What do you mean by "this is a test-only function"? I went up the chain that calls `mkInternals` and, if I'm not mistaken, it is transitively called from `LedgerDB.openDB` which is the primary entry point into the LedgerDB.

Reply:
It is created there, but the internals are ignored on production. See `f` in:
```
doOpenDB args initDb stream replayGoal =
    f <$> openDBInternal args initDb stream replayGoal
  where f (ldb, replayCounter, _) = (ldb, replayCounter)
```